### PR TITLE
Fix out res

### PR DESCRIPTION
**Pull request by vyepez88**

### CODE REVIEW

**Comment by vyepez88** [comment]
Simplifies the way OUTRIDER results are saved and the link to download them. 
Standardizes with FRASER.

File: drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R [hunk 1]
```diff
@@ -125,19 +125,13 @@ if (nrow(res) > 0) {
 
 
 #' ## Results table
-#+echo=F
+
+## Save results table in html folder and provide link to download
+file <- gsub(".html$", ".tsv", snakemake@output$wBhtml)
```
Comment (mumichae) [inline drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R]: Please create the output path in the Snakemake header, not here

File: drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R [hunk 2]
```diff
@@ -125,19 +125,13 @@ if (nrow(res) > 0) {
...
+fwrite(res, file, sep = '\t', quote = F)
+#+ echo=FALSE, results='asis'
+cat(paste0("<a href='./", basename(file), "'>Download OUTRIDER results table</a>"))
```
Comment (mumichae) [inline drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R:135]: Is it guaranteed that the download file is in the same directory as the html output? Maybe add a comment

File: drop/modules/aberrant-splicing-pipeline/FRASER/Summary.R [hunk 3]
```diff
@@ -93,15 +93,12 @@ for(type in psiTypes){
   after
 }
 
-#' # Results
+#' ## Results
 res <- fread(snakemake@input$results)
 file <- gsub(".html$", ".tsv", snakemake@output$wBhtml)
```
Comment (mumichae) [inline drop/modules/aberrant-splicing-pipeline/FRASER/Summary.R]: Same as for aberrant expression, please specify this path in the Snakemake header

**Review by mumichae — CHANGES_REQUESTED** [review]
Looks good overall, I trust that it works. Some minor comments on defining output file paths. It's good to have all outputs in the Snakemake header, so that they can be handled by Snakemake

File: drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R (hunk 2 above)
Comment (vyepez88) [reply drop/modules/aberrant-expression-pipeline/OUTRIDER/Summary.R:135]: it's guaranteed

**Review by mumichae — APPROVED** [review]

**Pull request merged**